pkg/didcomm/protocol/introduce/service.go

```diff
@@ -95,6 +100,9 @@ type Service struct {
 type Provider interface {
 	OutboundDispatcher() dispatcher.Outbound
 	StorageProvider() storage.Provider
+	// this method should be provided by the introduce client
+	// method should be implemented in didexchange Client
+	SendInvitation(inv *didexchange.Invitation, dest *service.Destination) error
```

Comment:
The services are initialized during framework creation by passing the context (eg didExchange service [initialization](https://github.com/hyperledger/aries-framework-go/blob/539fbb4c2c29535d2b531997425aaa2cc95420c1/pkg/framework/aries/default.go#L77)). At that point, I'm not sure we would have this dependency.

Reply:
@llorllale @troyronda @rolsonquadras  I am going to get it directly from the context, Are you ok with that?
```
svc, err := ctx.Service(didexchange.DIDExchange)
```